wagn/lib/wagn/application.rb

```diff
@@ -76,7 +76,10 @@ def paths
         paths["app/models"] = []
         paths["app/mailers"] = []
 
-        add_path paths, "config/routes.rb", with: "rails/application-routes.rb"
+        unless paths["config/routes.rb"].existent.present?
+          add_path paths, "config/routes.rb",
+                   with: "rails/application-routes.rb"
+        end
```

Comment:
I don't think this will solve the problem (although it's good to have).

The routes are supposed to be relative to the Rails root; if you change the rails root, nothing should have to change in the routes file.

I'll put more detail in the PT story.